### DISCUSSION TITLE
Not reading binaries on Laravel Sail

### DISCUSSION THREAD

**Discussion by badasukerubin** [spatie/laravel-image-optimizer#107] [Q&A]
I have installed all the binaries on my mac as described here: https://github.com/spatie/image-optimizer#optimization-tools.
When I try to upload an image, I get: `Process errored with sh: 1: jpegoptim: not found `

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/20546078/164728377-1d217e0b-d09e-4423-bb24-85c5c8810a16.png">

**Comment by lucasweaver10** [comment]
@badasukerubin Did you ever find a fix for this?
